sources/packages/backend/apps/api/src/esdc-integration/sin-validation/sin-validation-processing.service.ts

```diff
@@ -35,7 +30,7 @@ export class SINValidationProcessingService {
     private readonly sequenceService: SequenceControlService,
     private readonly sinValidationIntegrationService: SINValidationIntegrationService,
   ) {
-    this.esdcConfig = config.getConfig().ESDCIntegration;
+    this.esdcConfig = config.esdcIntegration;
```

Comment:
so we no longer have this `getConfig` in config service?

Reply:
I don't think that we need it. The problem with the getConfig is that it parses all the env variables, which is not needed for other applications like `Workers` or the queue consumers that will be introduced. That means that not all env variables will be present on all PODs, so, depending on how they are read, the missing variables can cause an issue just for not being present.